oracle/price-feeder/config.example.toml

```diff
@@ -55,7 +55,7 @@ enable_service_label = true
 enabled = true
 global_labels = [["chain-id", "sei-chain"]]
 service_name = "price-feeder"
-type = "prometheus"
+prometheus_retention = 60
```

Comment:
is this intended?

Reply:
yeah since the `type` field doesn't actully do anything